### PR TITLE
podman: run --replace prints only the new container id

### PR DESCRIPTION
**Pull request by giuseppe**
#### Does this PR introduce a user-facing change?

```release-note
run --replace prints only the ID for the new container, instead of the ID if the old one was stopped
```

### CODE REVIEW

**Review by Luap99 — APPROVED** [review]
LGTM

File: test/e2e/run_test.go [hunk 1]
```diff
@@ -1615,9 +1615,12 @@ VOLUME %s`, ALPINE, volPath, volPath)
 		// Run and replace 5 times in a row the "same" container.
 		ctrName := "testCtr"
 		for i := 0; i < 5; i++ {
-			session := podmanTest.Podman([]string{"run", "--detach", "--replace", "--name", ctrName, ALPINE, "/bin/sh"})
+			session := podmanTest.Podman([]string{"run", "--detach", "--replace", "--name", ctrName, ALPINE, "top"})
 			session.WaitWithDefaultTimeout()
 			Expect(session).Should(ExitCleanly())
+
+			// make sure Podman prints only one ID
+			Expect(session.OutputToStringArray()).To(HaveLen(1))
```
Comment (vrothberg) [inline test/e2e/run_test.go:1623]: Nit: In case you have to repush, I'd like to check that it's really the CID.

**Review by vrothberg — APPROVED** [review]
/lgtm
/hold

File: test/e2e/run_test.go (hunk 1 above)
Comment (edsantiago) [reply test/e2e/run_test.go:1623]: I'll second this. Very easy fix:
```diff
diff --git a/test/e2e/run_test.go b/test/e2e/run_test.go
index 036cd4569..423c2e44c 100644
--- a/test/e2e/run_test.go
+++ b/test/e2e/run_test.go
@@ -1622,3 +1622,3 @@ VOLUME %s`, ALPINE, volPath, volPath)
 			// make sure Podman prints only one ID
-			Expect(session.OutputToStringArray()).To(HaveLen(1))
+			Expect(session.OutputToString()).To(HaveLen(64))
 		}
```

File: test/e2e/run_test.go [hunk 2]
```diff
@@ -1615,9 +1615,12 @@ VOLUME %s`, ALPINE, volPath, volPath)
 		// Run and replace 5 times in a row the "same" container.
 		ctrName := "testCtr"
 		for i := 0; i < 5; i++ {
-			session := podmanTest.Podman([]string{"run", "--detach", "--replace", "--name", ctrName, ALPINE, "/bin/sh"})
+			session := podmanTest.Podman([]string{"run", "--detach", "--replace", "--name", ctrName, ALPINE, "top"})
 			session.WaitWithDefaultTimeout()
 			Expect(session).Should(ExitCleanly())
+
+			// make sure Podman prints only one ID
+			Expect(session.OutputToStringArray()).To(HaveLen(64))
```
Comment (edsantiago) [inline test/e2e/run_test.go:1623]: Oops, I think this is going to fail - it should be `OutputToString()`, not `...Array()`

**Comment by TomSweeneyRedHat** [comment]
LGTM
once the test is adjuated

**Pull request merged**